packages/playground-examples/copy/ja/TypeScript/Primitives/Unknown and Never.ts

```diff
@@ -1,14 +1,14 @@
-// Unknown
+// unknown型
 
-// Unknown is one of those types that once it clicks, you
-// can find quite a lot of uses for it. It acts like a sibling
-// to the any type. Where any allows for ambiguity - unknown
-// requires specifics.
+// unknown型は、非常に多くの箇所で使用されています。
+// unknown型は、any型とよく似た振る舞いをします。
```

Comment:
clickには`if two people click, they like, understand, and agree with each other`というように仲良くなるという用法があるらしいです
おそらく、この用法だと思われます
https://www.ldoceonline.com/jp/dictionary/click
その上で、usesは使われている場所よりも、用途という意味に思えるので

```suggestion
// unknown型は、一度詳しくなると、非常に多くの用途があることに気が付きます。
// unknown型は、any型とよく似た振る舞いをします。
```

Reply:
ありがとうございます、clickにそのような意味があるのですね :bow:
そうしますと、一応 `one of those types` の部分も訳して↓のような形にしますか？

```suggestion
// unknown型は、一度詳しくなると、非常に多くの用途があることに気が付く型の一つです。
// unknown型は、any型とよく似た振る舞いをします。
```

Reply:
👍